diff --git a/src/App.css b/src/App.css
index 74b5e05..e70a402 100644
--- a/src/App.css
+++ b/src/App.css
@@ -35,4 +35,4 @@
   to {
     transform: rotate(360deg);
   }
-}
+}
\ No newline at end of file
diff --git a/src/App.js b/src/App.js
index 3784575..509d47b 100644
--- a/src/App.js
+++ b/src/App.js
@@ -1,23 +1,29 @@
-import logo from './logo.svg';
-import './App.css';
+// export default App;
+import { useState } from "react";
+import { shuffleCards } from "./utils/shuffleCards";
+import { uniqueCards } from "./utils/uniqueCards";
+import Card from "./Components/Cards";
 
 function App() {
+  const [cards, setCards] = useState(() =>
+    shuffleCards(uniqueCards.concat(uniqueCards))
+  );
+
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div className="App" style={{display: 'flex', alignItems: 'center', margin: '30px auto', height: '85vh', width: '720px', backgroundColor: '#f1f2f3', padding: '15px 30px'}}>
+      <div style={{display: 'flex', flexWrap: 'wrap', gap: '15px', padding: '15px 30px' }}>
+        {cards.map((card, index) => {
+          return (
+            <Card
+            />
+          );
+        })}
+      </div>
+      <div style={{minWidth: '200px', height: '275px', margin: '15px 0'}}>
+        <div style={{backgroundColor: '#fff', padding: '15px 30px', height: '275px',}}>
+
+        </div>
+      </div>
     </div>
   );
 }
diff --git a/src/Components/Cards.jsx b/src/Components/Cards.jsx
new file mode 100644
index 0000000..cb8b4e1
--- /dev/null
+++ b/src/Components/Cards.jsx
@@ -0,0 +1,18 @@
+import React from 'react';
+import frontImage from '../assets/images/FrontImage.jpeg'
+console.log(frontImage,"fnt");
+
+const Card = () => {
+
+  return (
+    <div
+    >
+      <div style={{display: 'flex', alignItems: 'center', justifyContent: 'center', color:'#fff', width: "80px", height: '80px', backgroundColor: '#0095f5', borderRadius: '5px'}} className="card-face card-font-face">
+        {/* <img src={frontImage} alt="frontImage"  style={{width: "100%", height: '100%'}} /> */}
+        <p>?</p>
+      </div>
+    </div>
+  );
+};
+
+export default Card;
\ No newline at end of file
diff --git a/src/index.css b/src/index.css
index ec2585e..fb05047 100644
--- a/src/index.css
+++ b/src/index.css
@@ -5,9 +5,11 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+   
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
 }
+
